tests/helpers.ts

```diff
@@ -0,0 +1,17 @@
+import { test as base, expect } from '@grafana/plugin-e2e';
+import { QueryEditorPage } from './queryEditor.page';
+
+interface Fixtures {
+  queryEditor: QueryEditorPage;
+}
+
+const test = base.extend<Fixtures>({
+  /** Isolated `QueryEditorPage` instance. */
+  queryEditor: async ({ page, panelEditPage }, use) => {
+    const queryEditor = new QueryEditorPage({ page, panelEditPage });
+
+    await use(queryEditor);
+  },
+});
+
+export { test, expect };
```

Comment:
hmmm.... Maybe this is a personal style preference and a lack of familiarity with playwright conventions but I am not sure this abstraction is easier to read/write tests with? It definitely dries up the code, but if a test does fail or you need to write your own test you now need to know to look in a helper file to find this fixture and then look at queryeditorpage and then there might be a worry that a small change in one could affect other tests?

I guess when it comes to tests I almost don't mind a bit of redundancy if it makes them easier to read. But I don't want to block for a personal preference. 

Idk what do you think @kevinwcyu in your research does this kind of best practice for playwright you think? I'm happy to go with whatever you all think makes the most sense.

Reply:
Definitely, makes sense. I'm a fan of using fixtures, but this is indeed a very simple fixture and does not currently add much value beyond preventing you from having to instantiate the page object model.

As the test cases are expanded, something which may add a lot of value are test fixtures which reduce setup boilerplate when we're trying to test a particular state of the query editor multiple times. For instance, a `getPropertyValueQueryEditor` providing a query editor with the "Get property value" query pre-selected and exposing a page object model with the locators for form elements specific to only that query (i.e., no resolution/aggregation/etc.). Or even more valuable would be a fixture with an asset property (or multiple) already selected so that you can perform a series a tests without explicitly going through the flow of selecting an asset and then selecting a property in every test.

But those more valuable features don't need what I did in this PR now lol. If we want to wait until the value is clear, I am fine removing the fixture from this change. :)

Reply:
I think this structure provides a good foundation for code standard/style 👍 We might not see the value on the current scale but doesn't hurt to keep it

Reply:
I think it's useful to include this for interacting with the different select elements in the query editor, especially for the select elements that load options asynchronously. I'd be happy to keep it in since it does follow Playwright conventions.

Reply:
Ok cool, let's give it a go, and see how it works out for us!